.ci/integration_test.sh

```diff
@@ -373,7 +373,7 @@ integration_gm()
     LOG_INFO "integration_gm testing..."
     execute_cmd "bash tools/download_solc.sh -v 0.8.11 -g"
 
-    bash build_chain.sh -l 127.0.0.1:2 -s -o nodes_gm
+    bash build_chain.sh -l 127.0.0.1:2 -s -o nodes_gm -a 0x83309d045a19c44dc3722d15a6abd472f95866ac
```

Comment:
```suggestion
    bash build_chain.sh -l 127.0.0.1:2 -s -o nodes_gm -a 0x791a0073e6dfd9dc5e5061aebc43ab4f7aa4ae8b
```
国密要用国密的帐户，不能混用非国密的帐户